src/router/index.tsx

```diff
@@ -23,12 +23,12 @@ export const AppRouter = () => {
 
   return (
     <Switch>
-      <Route exact path="/" component={AuctionsView} />
-      <Route exact path="/auctions" component={AuctionsView} />
-      <Route exact path="/auctions/:auctionId" component={AuctionView} />
-      <Route exact path="/auctions/fixed/:auctionId" component={FixedPriceAuctionView} />
+      <Route exact path="/" component={SalesView} />
+      <Route exact path="/sales" component={SalesView} />
+      <Route exact path="/sales/:saleId" component={SaleView} />
+      <Route exact path="/sales/fixed/:saleId" component={FixedPriceSaleView} />
       <Route exact path="/about" component={StaticView} />
-      <Route exact path="/contact" component={AuctionsView} />
+      <Route exact path="/contact" component={SalesView} />
```

Comment:
`/contact` should use `ContactView`. It seems like this chore was executed by replacing every `Auction` keyword with `Sale`. This could have undesirable effects.